### ISSUE TITLE
Write Memory Chiplet constraints in AirScript

### ISSUE THREAD

**Issue by Overcastan** [0xPolygonMiden/air-script#182]

**Comment by bobbinth** [comment]
I believe this is currently blocked by https://github.com/0xMiden/miden-vm/issues/1614.

**Comment by Al-Kindi-0** [comment]
Indeed, should we prioritize it then ?

**Comment by bobbinth** [comment]
> Indeed, should we prioritize it then ?

I think so. I believe you already had a proposal for how to solve it - it just needs to be reviewed, right?

**Comment by Al-Kindi-0** [comment]
Yes, and I think the solution should be implemented as well to make sure we haven't missed anything which basically means doing the range checks and sending them through the ACE wiring bus.
